.phpcs.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<ruleset name="Project">
```

Comment:
Why do you want to add PHPCodeSniffer while [PHP CS Fixer](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer) is configured in the project ?

I don't think that is a good idea to add two different projects for the same thing.